<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Force State:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_force">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Virtual Force</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>725</width>
     <height>160</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btn_resetFault">
        <property name="text">
         <string>CMD_ResetFault</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Idle">
        <property name="text">
         <string>CMD_IDLE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_working">
        <property name="text">
         <string>CMD_Working</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_setFault">
        <property name="text">
         <string>CMD_SetFault</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <widget class="QCheckBox" name="cbx_axis_0">
        <property name="text">
         <string>Force_X</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="cbx_axis_1">
        <property name="text">
         <string>Force_Y</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="cbx_axis_2">
        <property name="text">
         <string>Force_Z</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="cbx_axis_3">
        <property name="text">
         <string>Force_a</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="cbx_axis_4">
        <property name="text">
         <string>Force_b</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="cbx_axis_5">
        <property name="text">
         <string>Force_c</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit_1"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QFormLayout" name="layout_monitor">
      <item row="0" column="1">
       <widget class="QLineEdit" name="theta_axis_0"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>AXIS_1:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="theta_axis_1"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>AXIS_2:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="theta_axis_2"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>AXIS_3:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="theta_axis_3"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>AXIS_4：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="theta_axis_4"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>AXIS_5:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="theta_axis_5"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>AXIS_0：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lab_force_state">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>20</y>
     <width>551</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>192.168.110.2: 8811</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_connect">
      <property name="text">
       <string>connect</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_disconnect">
      <property name="text">
       <string>disconnect</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lab_info">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
